.goreleaser.yaml

```
@@ -180,7 +180,7 @@ archives:
       {{- if .Arm }}v{{ .Arm }}{{ end }}
     format_overrides:
       - goos: windows
-        format: zip
+        formats: [ 'zip' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)